packages/crypto/src/transactions/types/htlc-lock.ts

```diff
@@ -25,7 +25,7 @@ export class HtlcLockTransaction extends Transaction {
     public serialize(options?: ISerializeOptions): ByteBuffer {
         const { data } = this;
 
-        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 4 + 8 + 21, true);
+        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 1 + 8 + 21, true);
```

Comment:
Isn't the expiration value only 4 bytes?

```suggestion
        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 1 + 4 + 21, true);
```

Reply:
Yes, good catch.

Reply:
The same goes for https://github.com/ArkEcosystem/java-crypto/pull/56 then.

Reply:
nice, thanks